src/prog_models/linear_model.py

```diff
@@ -0,0 +1,84 @@
+# Copyright © 2021 United States Government as represented by the Administrator of the
+# National Aeronautics and Space Administration.  All Rights Reserved.
+
+from . import PrognosticsModel
+from abc import ABC, abstractmethod
+import numpy as np
+
+
+class LinearModel(PrognosticsModel, ABC):
+    """
+    A linear prognostics model. Used when behavior can be described using a simple linear time-series model defined by the following equations:
+        * dx/dt = Ax + Bu + C
```

Comment:
Notation: typical notation is sligthly different. I'd recommend following the standard one, and I can rewrite LinearModel accordingly.

dx/dt = Ax + Bu + E
y = Cx + D
es = Fx + G